### PR TITLE
add some minor fixes and description about seting up mythaistar app

### PR DESCRIPTION
**Pull request by other019**

### CODE REVIEW

File: documentation/MrChecker-download/MyThaiStar.asciidoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
+= My Thai Star application setup
+My Thai Star is a reference application for DevonFW so it was used extensively in majority of our examples. To make them run properly you definitely should set it up somewhere and configure environment.csv acordingly.
```
Comment (Pyskaczu) [inline documentation/MrChecker-download/MyThaiStar.asciidoc:19]: accordingly

File: documentation/MrChecker-download/MyThaiStar.asciidoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+You can get the app from it official repository here https://github.com/devonfw/my-thai-star.
```
Comment (Pyskaczu) [inline documentation/MrChecker-download/MyThaiStar.asciidoc:20]: its official

**Review by Pyskaczu — COMMENTED** [review]
Fix and merge.

**Comment by other019** [comment]
@Pyskaczu corrected

File: documentation/MrChecker-download/MyThaiStar.asciidoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+== Setting up My Thai Start app
+Most of the important informations are covered in https://github.com/devonfw/my-thai-star#deployment.
+
+.The quick summary would be:
+. Get the machine with docker and docker-compose
+. Download the repository
+. Run docker-compose up
+. Go to your project to set up envrionment.csv
+ . The variables we are interested in are MY_THAI_STAR_URL and MY_THAI_STAR_API_URL
+ . If you set up My Thai Star on different host adjust the values accordingly
```
Comment (Pyskaczu) [inline documentation/MrChecker-download/MyThaiStar.asciidoc:31]: Add url to GUI and API according to: https://github.com/devonfw/mrchecker/issues/356

**Review by Pyskaczu — APPROVED** [review]

**Pull request merged**